docs/manual/en/manual.txt

```diff
@@ -0,0 +1,258 @@
+// -*- mode: adoc; -*-
+
+:lang: en
+:toc-title: Table of Contents
+:last-update-label: Last update
+:warning-caption: Attention
+
+= User's Guide
+Jean Pierre Cimalando, Olivier Humbert, Bruce Sutherland
+v1.1
+:toc:
+
+[.Lead]
+This manual documents the use of ADLplug, a frequency modulation (FM) synthesizer based on the technology of the OPL3. +
+ADLplug is free and open source software. +
+If you encounter a problem with the software, please create an issue on the project page: +
+https://github.com/jpcima/ADLplug
+
+== Description
+
+ADLplug is an FM syntheiszer, built on Yamaha's OPL3 standard.
+The implementation of this synthesis is based on an accurate emulation of the YMF262 chip.
```

Comment:
It looks like OPL3 and YMF262 are used interchangeably in the manual. Is it worth creating a glossary to explain that they are the same thing?

Reply:
Sounds a good idea, maybe just a line at the beginning of the doc ?